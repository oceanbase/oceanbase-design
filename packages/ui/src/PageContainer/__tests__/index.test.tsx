import React from 'react';
import { render } from '@testing-library/react';
import BasicDemo from '../demo/basic';
import ExtraAndFooterDemo from '../demo/extra-footer';
import TitleCompatibleDemo from '../demo/title';

describe('PageContainer', () => {
  it('basic', () => {
    const { asFragment } = render(<BasicDemo />);
    expect(asFragment().firstChild).toMatchSnapshot();
  });

  it('extra and footer', () => {
    const { asFragment } = render(<ExtraAndFooterDemo />);
    expect(asFragment().firstChild).toMatchSnapshot();
  });

  it('title', () => {
    const { asFragment } = render(<TitleCompatibleDemo />);
    expect(asFragment().firstChild).toMatchSnapshot();
  });
});
