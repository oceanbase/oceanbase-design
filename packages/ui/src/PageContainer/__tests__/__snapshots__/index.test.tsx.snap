// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`PageContainer > basic 1`] = `
<div
  class="ant-pro-page-container ant-pro-page-container-with-footer"
>
  <div
    class="ant-page-header ant-pro-page-container-warp-page-header ant-page-header-has-breadcrumb ant-page-header-ghost"
  >
    <nav
      class="ant-breadcrumb ant-page-header-breadcrumb"
    >
      <ol>
        <li>
          <a>
            一级页面
          </a>
        </li>
        <li
          aria-hidden="true"
          class="ant-breadcrumb-separator"
        >
          /
        </li>
        <li>
          <a>
            二级页面
          </a>
        </li>
        <li
          aria-hidden="true"
          class="ant-breadcrumb-separator"
        >
          /
        </li>
        <li>
          <span>
            当前页面
          </span>
        </li>
      </ol>
    </nav>
    <div
      class="ant-page-header-heading "
    >
      <div
        class="ant-page-header-heading-left"
      >
        <span
          class="ant-page-header-heading-title"
          title="页面标题"
        >
          页面标题
        </span>
        <span
          class="ant-page-header-heading-sub-title"
        >
          <div
            class="ant-space ant-space-horizontal ant-space-align-center ant-space-not-support-gap"
            style="column-gap: 12px; row-gap: 12px;"
          >
            <div
              class="ant-space-item"
            >
              <span
                aria-label="sync"
                class="anticon anticon-sync ant-page-header-heading-reload"
                role="img"
                tabindex="-1"
              >
                <svg
                  aria-hidden="true"
                  data-icon="sync"
                  fill="currentColor"
                  focusable="false"
                  height="1em"
                  viewBox="64 64 896 896"
                  width="1em"
                >
                  <path
                    d="M168 504.2c1-43.7 10-86.1 26.9-126 17.3-41 42.1-77.7 73.7-109.4S337 212.3 378 195c42.4-17.9 87.4-27 133.9-27s91.5 9.1 133.8 27A341.5 341.5 0 01755 268.8c9.9 9.9 19.2 20.4 27.8 31.4l-60.2 47a8 8 0 003 14.1l175.7 43c5 1.2 9.9-2.6 9.9-7.7l.8-180.9c0-6.7-7.7-10.5-12.9-6.3l-56.4 44.1C765.8 155.1 646.2 92 511.8 92 282.7 92 96.3 275.6 92 503.8a8 8 0 008 8.2h60c4.4 0 7.9-3.5 8-7.8zm756 7.8h-60c-4.4 0-7.9 3.5-8 7.8-1 43.7-10 86.1-26.9 126-17.3 41-42.1 77.8-73.7 109.4A342.45 342.45 0 01512.1 856a342.24 342.24 0 01-243.2-100.8c-9.9-9.9-19.2-20.4-27.8-31.4l60.2-47a8 8 0 00-3-14.1l-175.7-43c-5-1.2-9.9 2.6-9.9 7.7l-.7 181c0 6.7 7.7 10.5 12.9 6.3l56.4-44.1C258.2 868.9 377.8 932 512.2 932c229.2 0 415.5-183.7 419.8-411.8a8 8 0 00-8-8.2z"
                  />
                </svg>
              </span>
            </div>
          </div>
        </span>
      </div>
      <span
        class="ant-page-header-heading-extra"
      >
        <div
          class="ant-space ant-space-horizontal ant-space-align-center ant-space-gap-row-small ant-space-gap-col-small"
        >
          <div
            class="ant-space-item"
          >
            <button
              class="ant-btn ant-btn-default"
              type="button"
            >
              <span>
                次要按钮
              </span>
            </button>
          </div>
          <div
            class="ant-space-item"
          >
            <button
              class="ant-btn ant-btn-primary"
              type="button"
            >
              <span>
                主要按钮
              </span>
            </button>
          </div>
          <div
            class="ant-space-item"
          >
            <button
              class="ant-btn ant-btn-default ant-dropdown-trigger"
              style="padding: 0px 8px;"
              type="button"
            >
              <span
                aria-label="ellipsis"
                class="anticon anticon-ellipsis"
                role="img"
              >
                <svg
                  aria-hidden="true"
                  data-icon="ellipsis"
                  fill="currentColor"
                  focusable="false"
                  height="1em"
                  viewBox="64 64 896 896"
                  width="1em"
                >
                  <path
                    d="M176 511a56 56 0 10112 0 56 56 0 10-112 0zm280 0a56 56 0 10112 0 56 56 0 10-112 0zm280 0a56 56 0 10112 0 56 56 0 10-112 0z"
                  />
                </svg>
              </span>
            </button>
          </div>
        </div>
      </span>
    </div>
  </div>
  <div
    class="ant-pro-grid-content"
  >
    <div
      class="ant-pro-grid-content-children"
    >
      <div
        class="ant-pro-page-container-children-container"
      >
        <div
          class="ant-descriptions"
        >
          <div
            class="ant-descriptions-view"
          >
            <table>
              <tbody>
                <tr
                  class="ant-descriptions-row"
                >
                  <td
                    class="ant-descriptions-item"
                    colspan="1"
                  >
                    <div
                      class="ant-descriptions-item-container"
                    >
                      <span
                        class="ant-descriptions-item-label"
                      >
                        创建人
                      </span>
                      <span
                        class="ant-descriptions-item-content"
                      >
                        <span
                          class="ant-typography ant-typography-ellipsis ant-typography-single-line ant-typography-ellipsis-single-line"
                        >
                          曲丽丽
                        </span>
                      </span>
                    </div>
                  </td>
                  <td
                    class="ant-descriptions-item"
                    colspan="1"
                  >
                    <div
                      class="ant-descriptions-item-container"
                    >
                      <span
                        class="ant-descriptions-item-label"
                      >
                        电话号码
                      </span>
                      <span
                        class="ant-descriptions-item-content"
                      >
                        <span
                          class="ant-typography ant-typography-ellipsis ant-typography-single-line ant-typography-ellipsis-single-line"
                        >
                          1810000000
                        </span>
                      </span>
                    </div>
                  </td>
                  <td
                    class="ant-descriptions-item"
                    colspan="1"
                  >
                    <div
                      class="ant-descriptions-item-container"
                    >
                      <span
                        class="ant-descriptions-item-label"
                      >
                        地址
                      </span>
                      <span
                        class="ant-descriptions-item-content"
                      >
                        <span
                          class="ant-typography ant-typography-ellipsis ant-typography-single-line ant-typography-ellipsis-single-line"
                        >
                          浙江省杭州市西湖区工专路
                        </span>
                      </span>
                    </div>
                  </td>
                </tr>
                <tr
                  class="ant-descriptions-row"
                >
                  <td
                    class="ant-descriptions-item"
                    colspan="1"
                  >
                    <div
                      class="ant-descriptions-item-container"
                    >
                      <span
                        class="ant-descriptions-item-label"
                      >
                        关联表单
                      </span>
                      <span
                        class="ant-descriptions-item-content"
                      >
                        <span
                          class="ant-typography ant-typography-ellipsis ant-typography-single-line ant-typography-ellipsis-single-line"
                        >
                          <a>
                            421421
                          </a>
                        </span>
                      </span>
                    </div>
                  </td>
                  <td
                    class="ant-descriptions-item"
                    colspan="1"
                  >
                    <div
                      class="ant-descriptions-item-container"
                    >
                      <span
                        class="ant-descriptions-item-label"
                      >
                        创建时间
                      </span>
                      <span
                        class="ant-descriptions-item-content"
                      >
                        <span
                          class="ant-typography ant-typography-ellipsis ant-typography-single-line ant-typography-ellipsis-single-line"
                        >
                          2017-01-10
                        </span>
                      </span>
                    </div>
                  </td>
                  <td
                    class="ant-descriptions-item"
                    colspan="1"
                  >
                    <div
                      class="ant-descriptions-item-container"
                    >
                      <span
                        class="ant-descriptions-item-label"
                      >
                        备注
                      </span>
                      <span
                        class="ant-descriptions-item-content"
                      >
                        <span
                          class="ant-typography ant-typography-ellipsis ant-typography-single-line ant-typography-ellipsis-single-line"
                        >
                          这是备注
                        </span>
                      </span>
                    </div>
                  </td>
                </tr>
              </tbody>
            </table>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`PageContainer > extra and footer 1`] = `
<div
  class="ant-pro-page-container ant-pro-page-container-with-footer"
>
  <div
    class="ant-page-header ant-pro-page-container-warp-page-header ant-page-header-has-breadcrumb ant-page-header-ghost"
  >
    <nav
      class="ant-breadcrumb ant-page-header-breadcrumb"
    >
      <ol>
        <li>
          <a>
            一级页面
          </a>
        </li>
        <li
          aria-hidden="true"
          class="ant-breadcrumb-separator"
        >
          /
        </li>
        <li>
          <a>
            二级页面
          </a>
        </li>
        <li
          aria-hidden="true"
          class="ant-breadcrumb-separator"
        >
          /
        </li>
        <li>
          <span>
            当前页面
          </span>
        </li>
      </ol>
    </nav>
    <div
      class="ant-page-header-heading "
    >
      <div
        class="ant-page-header-heading-left"
      >
        <span
          class="ant-page-header-heading-title"
          title="页面标题"
        >
          页面标题
        </span>
        <span
          class="ant-page-header-heading-sub-title"
        >
          <div
            class="ant-space ant-space-horizontal ant-space-align-center ant-space-not-support-gap"
            style="column-gap: 12px; row-gap: 12px;"
          >
            <div
              class="ant-space-item"
            >
              <span
                aria-label="sync"
                class="anticon anticon-sync ant-page-header-heading-reload"
                role="img"
                tabindex="-1"
              >
                <svg
                  aria-hidden="true"
                  data-icon="sync"
                  fill="currentColor"
                  focusable="false"
                  height="1em"
                  viewBox="64 64 896 896"
                  width="1em"
                >
                  <path
                    d="M168 504.2c1-43.7 10-86.1 26.9-126 17.3-41 42.1-77.7 73.7-109.4S337 212.3 378 195c42.4-17.9 87.4-27 133.9-27s91.5 9.1 133.8 27A341.5 341.5 0 01755 268.8c9.9 9.9 19.2 20.4 27.8 31.4l-60.2 47a8 8 0 003 14.1l175.7 43c5 1.2 9.9-2.6 9.9-7.7l.8-180.9c0-6.7-7.7-10.5-12.9-6.3l-56.4 44.1C765.8 155.1 646.2 92 511.8 92 282.7 92 96.3 275.6 92 503.8a8 8 0 008 8.2h60c4.4 0 7.9-3.5 8-7.8zm756 7.8h-60c-4.4 0-7.9 3.5-8 7.8-1 43.7-10 86.1-26.9 126-17.3 41-42.1 77.8-73.7 109.4A342.45 342.45 0 01512.1 856a342.24 342.24 0 01-243.2-100.8c-9.9-9.9-19.2-20.4-27.8-31.4l60.2-47a8 8 0 00-3-14.1l-175.7-43c-5-1.2-9.9 2.6-9.9 7.7l-.7 181c0 6.7 7.7 10.5 12.9 6.3l56.4-44.1C258.2 868.9 377.8 932 512.2 932c229.2 0 415.5-183.7 419.8-411.8a8 8 0 00-8-8.2z"
                  />
                </svg>
              </span>
            </div>
          </div>
        </span>
      </div>
      <span
        class="ant-page-header-heading-extra"
      >
        <div
          class="ant-space ant-space-horizontal ant-space-align-center ant-space-gap-row-small ant-space-gap-col-small"
        >
          <div
            class="ant-space-item"
          >
            <div
              class="ant-picker ant-picker-large ant-picker-outlined"
            >
              <div
                class="ant-picker-input"
              >
                <input
                  aria-invalid="false"
                  autocomplete="off"
                  placeholder="DatePicker"
                  size="12"
                  value=""
                />
                <span
                  class="ant-picker-suffix"
                >
                  <span
                    aria-label="calendar"
                    class="anticon anticon-calendar"
                    role="img"
                  >
                    <svg
                      aria-hidden="true"
                      data-icon="calendar"
                      fill="currentColor"
                      focusable="false"
                      height="1em"
                      viewBox="64 64 896 896"
                      width="1em"
                    >
                      <path
                        d="M880 184H712v-64c0-4.4-3.6-8-8-8h-56c-4.4 0-8 3.6-8 8v64H384v-64c0-4.4-3.6-8-8-8h-56c-4.4 0-8 3.6-8 8v64H144c-17.7 0-32 14.3-32 32v664c0 17.7 14.3 32 32 32h736c17.7 0 32-14.3 32-32V216c0-17.7-14.3-32-32-32zm-40 656H184V460h656v380zM184 392V256h128v48c0 4.4 3.6 8 8 8h56c4.4 0 8-3.6 8-8v-48h256v48c0 4.4 3.6 8 8 8h56c4.4 0 8-3.6 8-8v-48h128v136H184z"
                      />
                    </svg>
                  </span>
                </span>
              </div>
            </div>
          </div>
          <div
            class="ant-space-item"
          >
            <div
              class="ant-picker ant-picker-range ant-picker-large ant-picker-outlined"
            >
              <div
                class="ant-picker-input"
              >
                <input
                  aria-invalid="false"
                  autocomplete="off"
                  date-range="start"
                  placeholder="开始日期"
                  size="12"
                  value=""
                />
              </div>
              <div
                class="ant-picker-range-separator"
              >
                <span
                  aria-label="to"
                  class="ant-picker-separator"
                >
                  <span
                    aria-label="swap-right"
                    class="anticon anticon-swap-right"
                    role="img"
                  >
                    <svg
                      aria-hidden="true"
                      data-icon="swap-right"
                      fill="currentColor"
                      focusable="false"
                      height="1em"
                      viewBox="0 0 1024 1024"
                      width="1em"
                    >
                      <path
                        d="M873.1 596.2l-164-208A32 32 0 00684 376h-64.8c-6.7 0-10.4 7.7-6.3 13l144.3 183H152c-4.4 0-8 3.6-8 8v60c0 4.4 3.6 8 8 8h695.9c26.8 0 41.7-30.8 25.2-51.8z"
                      />
                    </svg>
                  </span>
                </span>
              </div>
              <div
                class="ant-picker-input"
              >
                <input
                  aria-invalid="false"
                  autocomplete="off"
                  date-range="end"
                  placeholder="结束日期"
                  size="12"
                  value=""
                />
              </div>
              <div
                class="ant-picker-active-bar"
                style="position: absolute; width: 0px; left: 0px;"
              />
              <span
                class="ant-picker-suffix"
              >
                <span
                  aria-label="calendar"
                  class="anticon anticon-calendar"
                  role="img"
                >
                  <svg
                    aria-hidden="true"
                    data-icon="calendar"
                    fill="currentColor"
                    focusable="false"
                    height="1em"
                    viewBox="64 64 896 896"
                    width="1em"
                  >
                    <path
                      d="M880 184H712v-64c0-4.4-3.6-8-8-8h-56c-4.4 0-8 3.6-8 8v64H384v-64c0-4.4-3.6-8-8-8h-56c-4.4 0-8 3.6-8 8v64H144c-17.7 0-32 14.3-32 32v664c0 17.7 14.3 32 32 32h736c17.7 0 32-14.3 32-32V216c0-17.7-14.3-32-32-32zm-40 656H184V460h656v380zM184 392V256h128v48c0 4.4 3.6 8 8 8h56c4.4 0 8-3.6 8-8v-48h256v48c0 4.4 3.6 8 8 8h56c4.4 0 8-3.6 8-8v-48h128v136H184z"
                    />
                  </svg>
                </span>
              </span>
            </div>
          </div>
          <div
            class="ant-space-item"
          >
            <div
              class="ant-picker ant-picker-large ant-picker-outlined"
            >
              <div
                class="ant-picker-input"
              >
                <input
                  aria-invalid="false"
                  autocomplete="off"
                  placeholder="TimePicker"
                  size="10"
                  value=""
                />
                <span
                  class="ant-picker-suffix"
                >
                  <span
                    aria-label="clock-circle"
                    class="anticon anticon-clock-circle"
                    role="img"
                  >
                    <svg
                      aria-hidden="true"
                      data-icon="clock-circle"
                      fill="currentColor"
                      focusable="false"
                      height="1em"
                      viewBox="64 64 896 896"
                      width="1em"
                    >
                      <path
                        d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm0 820c-205.4 0-372-166.6-372-372s166.6-372 372-372 372 166.6 372 372-166.6 372-372 372z"
                      />
                      <path
                        d="M686.7 638.6L544.1 535.5V288c0-4.4-3.6-8-8-8H488c-4.4 0-8 3.6-8 8v275.4c0 2.6 1.2 5 3.3 6.5l165.4 120.6c3.6 2.6 8.6 1.8 11.2-1.7l28.6-39c2.6-3.7 1.8-8.7-1.8-11.2z"
                      />
                    </svg>
                  </span>
                </span>
              </div>
            </div>
          </div>
          <div
            class="ant-space-item"
          >
            <span
              class="ant-input-group-wrapper ant-input-group-wrapper-lg ant-input-group-wrapper-outlined ant-input-search ant-input-search-large"
            >
              <span
                class="ant-input-wrapper ant-input-group"
              >
                <span
                  class="ant-input-affix-wrapper ant-input-affix-wrapper-lg ant-input-outlined"
                >
                  <input
                    class="ant-input ant-input-lg"
                    placeholder="placeholder"
                    type="text"
                    value=""
                  />
                  <span
                    class="ant-input-suffix"
                  >
                    <span
                      class="ant-input-clear-icon ant-input-clear-icon-hidden"
                      role="button"
                      tabindex="-1"
                    >
                      <span
                        aria-label="close-circle"
                        class="anticon anticon-close-circle"
                        role="img"
                      >
                        <svg
                          aria-hidden="true"
                          data-icon="close-circle"
                          fill="currentColor"
                          fill-rule="evenodd"
                          focusable="false"
                          height="1em"
                          viewBox="64 64 896 896"
                          width="1em"
                        >
                          <path
                            d="M512 64c247.4 0 448 200.6 448 448S759.4 960 512 960 64 759.4 64 512 264.6 64 512 64zm127.98 274.82h-.04l-.08.06L512 466.75 384.14 338.88c-.04-.05-.06-.06-.08-.06a.12.12 0 00-.07 0c-.03 0-.05.01-.09.05l-45.02 45.02a.2.2 0 00-.05.09.12.12 0 000 .07v.02a.27.27 0 00.06.06L466.75 512 338.88 639.86c-.05.04-.06.06-.06.08a.12.12 0 000 .07c0 .03.01.05.05.09l45.02 45.02a.2.2 0 00.09.05.12.12 0 00.07 0c.02 0 .04-.01.08-.05L512 557.25l127.86 127.87c.04.04.06.05.08.05a.12.12 0 00.07 0c.03 0 .05-.01.09-.05l45.02-45.02a.2.2 0 00.05-.09.12.12 0 000-.07v-.02a.27.27 0 00-.05-.06L557.25 512l127.87-127.86c.04-.04.05-.06.05-.08a.12.12 0 000-.07c0-.03-.01-.05-.05-.09l-45.02-45.02a.2.2 0 00-.09-.05.12.12 0 00-.07 0z"
                          />
                        </svg>
                      </span>
                    </span>
                  </span>
                </span>
                <span
                  class="ant-input-group-addon"
                >
                  <button
                    class="ant-btn ant-btn-default ant-btn-lg ant-btn-icon-only ant-input-search-button"
                    type="button"
                  >
                    <span
                      class="ant-btn-icon"
                    >
                      <span
                        aria-label="search"
                        class="anticon anticon-search"
                        role="img"
                      >
                        <svg
                          aria-hidden="true"
                          data-icon="search"
                          fill="currentColor"
                          focusable="false"
                          height="1em"
                          viewBox="64 64 896 896"
                          width="1em"
                        >
                          <path
                            d="M909.6 854.5L649.9 594.8C690.2 542.7 712 479 712 412c0-80.2-31.3-155.4-87.9-212.1-56.6-56.7-132-87.9-212.1-87.9s-155.5 31.3-212.1 87.9C143.2 256.5 112 331.8 112 412c0 80.1 31.3 155.5 87.9 212.1C256.5 680.8 331.8 712 412 712c67 0 130.6-21.8 182.7-62l259.7 259.6a8.2 8.2 0 0011.6 0l43.6-43.5a8.2 8.2 0 000-11.6zM570.4 570.4C528 612.7 471.8 636 412 636s-116-23.3-158.4-65.6C211.3 528 188 471.8 188 412s23.3-116.1 65.6-158.4C296 211.3 352.2 188 412 188s116.1 23.2 158.4 65.6S636 352.2 636 412s-23.3 116.1-65.6 158.4z"
                          />
                        </svg>
                      </span>
                    </span>
                  </button>
                </span>
              </span>
            </span>
          </div>
          <div
            class="ant-space-item"
          >
            <div
              class="ant-select ant-select-lg ant-select-outlined ant-select-single ant-select-show-arrow"
              style="width: 100px;"
            >
              <div
                class="ant-select-selector"
              >
                <span
                  class="ant-select-selection-search"
                >
                  <input
                    aria-autocomplete="list"
                    aria-controls="rc_select_TEST_OR_SSR_list"
                    aria-expanded="false"
                    aria-haspopup="listbox"
                    aria-owns="rc_select_TEST_OR_SSR_list"
                    autocomplete="off"
                    class="ant-select-selection-search-input"
                    id="rc_select_TEST_OR_SSR"
                    readonly=""
                    role="combobox"
                    style="opacity: 0;"
                    type="search"
                    unselectable="on"
                    value=""
                  />
                </span>
                <span
                  class="ant-select-selection-item"
                  title="large"
                >
                  large
                </span>
              </div>
              <span
                aria-hidden="true"
                class="ant-select-arrow"
                style="user-select: none;"
                unselectable="on"
              >
                <span
                  aria-label="down"
                  class="anticon anticon-down ant-select-suffix"
                  role="img"
                >
                  <svg
                    aria-hidden="true"
                    data-icon="down"
                    fill="currentColor"
                    focusable="false"
                    height="1em"
                    viewBox="64 64 896 896"
                    width="1em"
                  >
                    <path
                      d="M884 256h-75c-5.1 0-9.9 2.5-12.9 6.6L512 654.2 227.9 262.6c-3-4.1-7.8-6.6-12.9-6.6h-75c-6.5 0-10.3 7.4-6.5 12.7l352.6 486.1c12.8 17.6 39 17.6 51.7 0l352.6-486.1c3.9-5.3.1-12.7-6.4-12.7z"
                    />
                  </svg>
                </span>
              </span>
            </div>
          </div>
          <div
            class="ant-space-item"
          >
            <div
              class="ant-radio-group ant-radio-group-outline ant-radio-group-large"
            >
              <label
                class="ant-radio-button-wrapper"
              >
                <span
                  class="ant-radio-button"
                >
                  <input
                    class="ant-radio-button-input"
                    type="radio"
                    value="input"
                  />
                  <span
                    class="ant-radio-button-inner"
                  />
                </span>
                <span>
                  Input
                </span>
              </label>
              <label
                class="ant-radio-button-wrapper ant-radio-button-wrapper-checked"
              >
                <span
                  class="ant-radio-button ant-radio-button-checked"
                >
                  <input
                    checked=""
                    class="ant-radio-button-input"
                    type="radio"
                    value="search"
                  />
                  <span
                    class="ant-radio-button-inner"
                  />
                </span>
                <span>
                  Search
                </span>
              </label>
              <label
                class="ant-radio-button-wrapper"
              >
                <span
                  class="ant-radio-button"
                >
                  <input
                    class="ant-radio-button-input"
                    type="radio"
                    value="addon"
                  />
                  <span
                    class="ant-radio-button-inner"
                  />
                </span>
                <span>
                  Addon
                </span>
              </label>
            </div>
          </div>
          <div
            class="ant-space-item"
          >
            <button
              class="ant-btn ant-btn-primary ant-btn-lg"
              type="button"
            >
              <span>
                主要按钮
              </span>
            </button>
          </div>
          <div
            class="ant-space-item"
          >
            <button
              class="ant-btn ant-btn-default ant-btn-lg ant-dropdown-trigger"
              style="padding: 0px 8px;"
              type="button"
            >
              <span
                aria-label="ellipsis"
                class="anticon anticon-ellipsis"
                role="img"
              >
                <svg
                  aria-hidden="true"
                  data-icon="ellipsis"
                  fill="currentColor"
                  focusable="false"
                  height="1em"
                  viewBox="64 64 896 896"
                  width="1em"
                >
                  <path
                    d="M176 511a56 56 0 10112 0 56 56 0 10-112 0zm280 0a56 56 0 10112 0 56 56 0 10-112 0zm280 0a56 56 0 10112 0 56 56 0 10-112 0z"
                  />
                </svg>
              </span>
            </button>
          </div>
        </div>
      </span>
    </div>
  </div>
  <div
    class="ant-pro-grid-content"
  >
    <div
      class="ant-pro-grid-content-children"
    >
      <div
        class="ant-pro-page-container-children-container"
      >
        <div
          class="ant-descriptions"
        >
          <div
            class="ant-descriptions-view"
          >
            <table>
              <tbody>
                <tr
                  class="ant-descriptions-row"
                >
                  <td
                    class="ant-descriptions-item"
                    colspan="1"
                  >
                    <div
                      class="ant-descriptions-item-container"
                    >
                      <span
                        class="ant-descriptions-item-label"
                      >
                        创建人
                      </span>
                      <span
                        class="ant-descriptions-item-content"
                      >
                        <span
                          class="ant-typography ant-typography-ellipsis ant-typography-single-line ant-typography-ellipsis-single-line"
                        >
                          曲丽丽
                        </span>
                      </span>
                    </div>
                  </td>
                  <td
                    class="ant-descriptions-item"
                    colspan="1"
                  >
                    <div
                      class="ant-descriptions-item-container"
                    >
                      <span
                        class="ant-descriptions-item-label"
                      >
                        电话号码
                      </span>
                      <span
                        class="ant-descriptions-item-content"
                      >
                        <span
                          class="ant-typography ant-typography-ellipsis ant-typography-single-line ant-typography-ellipsis-single-line"
                        >
                          1810000000
                        </span>
                      </span>
                    </div>
                  </td>
                  <td
                    class="ant-descriptions-item"
                    colspan="1"
                  >
                    <div
                      class="ant-descriptions-item-container"
                    >
                      <span
                        class="ant-descriptions-item-label"
                      >
                        地址
                      </span>
                      <span
                        class="ant-descriptions-item-content"
                      >
                        <span
                          class="ant-typography ant-typography-ellipsis ant-typography-single-line ant-typography-ellipsis-single-line"
                        >
                          浙江省杭州市西湖区工专路
                        </span>
                      </span>
                    </div>
                  </td>
                </tr>
                <tr
                  class="ant-descriptions-row"
                >
                  <td
                    class="ant-descriptions-item"
                    colspan="1"
                  >
                    <div
                      class="ant-descriptions-item-container"
                    >
                      <span
                        class="ant-descriptions-item-label"
                      >
                        关联表单
                      </span>
                      <span
                        class="ant-descriptions-item-content"
                      >
                        <span
                          class="ant-typography ant-typography-ellipsis ant-typography-single-line ant-typography-ellipsis-single-line"
                        >
                          <a>
                            421421
                          </a>
                        </span>
                      </span>
                    </div>
                  </td>
                  <td
                    class="ant-descriptions-item"
                    colspan="1"
                  >
                    <div
                      class="ant-descriptions-item-container"
                    >
                      <span
                        class="ant-descriptions-item-label"
                      >
                        创建时间
                      </span>
                      <span
                        class="ant-descriptions-item-content"
                      >
                        <span
                          class="ant-typography ant-typography-ellipsis ant-typography-single-line ant-typography-ellipsis-single-line"
                        >
                          2017-01-10
                        </span>
                      </span>
                    </div>
                  </td>
                  <td
                    class="ant-descriptions-item"
                    colspan="1"
                  >
                    <div
                      class="ant-descriptions-item-container"
                    >
                      <span
                        class="ant-descriptions-item-label"
                      >
                        备注
                      </span>
                      <span
                        class="ant-descriptions-item-content"
                      >
                        <span
                          class="ant-typography ant-typography-ellipsis ant-typography-single-line ant-typography-ellipsis-single-line"
                        >
                          这是备注
                        </span>
                      </span>
                    </div>
                  </td>
                </tr>
              </tbody>
            </table>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
